### DISCUSSION TITLE
(SUGGESTION) Thruster effects

### DISCUSSION THREAD

**Discussion by arbiter3904** [tsunamayo/Starship-EVO#4027] [Suggestions]
Pretty self explanatory basically an effect for the thrusters like you see on fight jets when they use after-burner

Here's an example 
![image](https://user-images.githubusercontent.com/73545229/106336864-b2837a00-6287-11eb-94a0-95494a75b020.png)

**Comment by Garrett-C** [comment]
Would be cool but because you sometimes have to hide them in walls etc it could end up looking funky. There are also the side ones to consider. You would probably need some way to set which ones have effects.

**Comment by Dwarf-LordPangolin** [comment]
There used to be a thruster effect, but it was either removed or is only visible under very specific circumstances; only the light cast by thrusters on nearby surfaces is still readily visible. You can see what the old thruster effect looked like here, coming out of the forward thrusters:

![20200509161647_1](https://user-images.githubusercontent.com/33041786/106349673-6c351780-6295-11eb-8b4e-9475c8961ba7.jpg)

I really miss the effect; it made the ships feel more alive and realistic. So I too would support this suggestion. 👍

**Comment by arbiter3904** [comment]
> "Would be cool but because you sometimes have to hide them in walls etc it could end up looking funky. There are also the side ones to consider. You would probably need some way to set which ones have effects."

Maybe there would be a way to toggle it on different engines

**Comment by cloudrambler** [comment]
What if the thrusters have two modes: reaction mode where they use fuel and generate an exhaust plume, and non-reaction mode (impulse) where they can be anywhere in the ship and just need power. The reaction thrusters could be more efficient/powerful than the impulse thrusters, to provide an advantage to ship designers that keep the engines external.

**Comment by TIKIRobo** [comment]
> What if the thrusters have two modes: reaction mode where they use fuel and generate an exhaust plume, and non-reaction mode (impulse) where they can be anywhere in the ship and just need power. The reaction thrusters could be more efficient/powerful than the impulse thrusters, to provide an advantage to ship designers that keep the engines external.

this would help a lot but then also people might just put the thrusters far enough inside their ship that they can't see the plumes but can get the extra speed

**Comment by cloudrambler** [comment]
> this would help a lot but then also people might just put the thrusters far enough inside their ship that they can't see the plumes but can get the extra speed

That could be done with the present thrusters; which I assume are intended to represent devices that generate physical "thrust" via expulsion of reaction mass, rather than directly providing momentum via quantum effects. So allowing us to add variety is not significantly changing the scope for ignoring the implicit thematic rules.

**Comment by Garrett-C** [comment]
The other thing to remember is on a very large ship you may need thruster that are more that 16m long. So you would have to have thrusters behind thrusters.
